nano/node/active_transactions.cpp

```diff
@@ -943,11 +943,11 @@ void nano::active_transactions::add_inactive_votes_cache (nano::block_hash const
 	// Check principal representative status
 	if (node.ledger.weight (representative_a) > node.minimum_principal_weight ())
 	{
-		auto existing (inactive_votes_cache.get<nano::gap_cache::tag_hash> ().find (hash_a));
-		if (existing != inactive_votes_cache.get<nano::gap_cache::tag_hash> ().end () && !existing->confirmed)
+		auto existing (inactive_votes_cache.get<tag_hash> ().find (hash_a));
```

Comment:
I've been cleaning up these verbose `inactive_votes_cache.get<nano::gap_cache::tag_hash> ()` when they are used many times in a function, with:
```cpp
auto & inactive_by_hash (inactive_votes_cache.get<nano::gap_cache::tag_hash> ());
auto existing (inactive_by_hash.find (hash_a));
```

This is one of the places that is most verbose, so could consider applying here. Maybe whenever there are at least 2 uses, same for tag_arrival